bpmn-model/src/test/java/io/zeebe/model/bpmn/builder/ProcessBuilderTest.java

```diff
@@ -1832,7 +1832,7 @@ public void testIntermediateConditionalEventDefinitionShortCut() {
         Bpmn.createProcess()
             .startEvent()
             .intermediateCatchEvent(CATCH_ID)
-            .condition(TEST_CONDITION)
+            .conditionExpression(TEST_CONDITION)
```

Comment:
Not sure what to make of this.

`TEST_CONDITION ="${true}";` 

Doesn't look like a FEEL expression to me. Might be a Json Path expression, but I thought we got rid of those?

If it is an expression, please rename the constant.

Reply:
Also, `BpmnTestConstants`could use some culling. My IDE tells me that many constants are not used anywhere

Reply:
This change was not intended. I reverted it.

This test class and most of the other stuff in the `bpmn-model` module (i.e. everything not prefixed with Zeebe) is forked from the Camunda model API. Let's not tough these classes :sweat_smile:  Keeping the fork in "sync" with the origin may help us to adopt changes in the upstream module.